draft out rate limit details (#3510)
camunda/camunda-docs: docs/apis-tools/administration-api/authentication.md

Access tokens have a validity period found in the access token. After this time, a new access token must be requested.
:::

> ## Built-in rate limit
> 
> The OAuth service has a built-in rate limit of about one request per second for all clients with the same source IP address.

Reviewer: @christinaausley it's a minor change, but I think I would suggest to completely remove the wording `built-in`: I'm not sure why it was there in the first place and it doesn't indicate anything really useful:

```suggestion
## Rate limit

The OAuth service rates limit all clients with the same source IP address to about one request per second.
```

Author: Done 👍
